### PR TITLE
Additional checks for AtVersion automation run support

### PR DESCRIPTION
**Pull request by howard-e**
Address #1226.

This PR explicitly checks for the AtVersion automation run support in `server/util/getAtVersionWithRequirements` with the already defined valid AtVersion constants instead of using the `AtVersion.supportedByAutomation` graphql attribute.

This is a quick patch due to the `supportedByAutomation` gql attribute not being returned as expected in this instance, while investigations into what may have changed continues.

Confirmed tests with this on the sandbox environment.

### CODE REVIEW

File: server/resources/ats.json [hunk 1]
```diff
@@ -55,10 +28,40 @@
...
+                    "If the virtual cursor is active, press &lt;kbd&gt;Insert&lt;/kbd&gt;+&lt;kbd&gt;z&lt;/kbd&gt; to disable the virtual cursor."
+                ]
+            }
+        }
+    },
     {
         "id": 3,
         "name": "VoiceOver for macOS",
         "key": "voiceover_macos",
+        "vendorId": 4,
```
Comment (jugglinmike) [inline server/resources/ats.json:64]: Why do we need to define the `vendorId` property for this patch? Would you mind if I moved JAWS back in order to make the diff clearer?

Comment (howard-e) [reply server/resources/ats.json:64]: Odd, just realized this happened. Looks to be a change included by #1208 that wasn't pushed. That can definitely be done separately

Comment (howard-e) [reply server/resources/ats.json]: Removed that change

Comment (jugglinmike) [reply server/resources/ats.json]: Awesome! I'll merge once CI passes

**Review by jugglinmike — APPROVED** [review]

**Pull request merged**